java/ql/src/semmle/code/java/frameworks/ApacheHttp.qll

```diff
@@ -13,3 +13,23 @@ class ApacheHttpEntityGetContent extends Method {
     this.getName() = "getContent"
   }
 }
+
+/**
+ *  A class derived from the `HttpRequestBase` or the `BasicHttpRequest`
+ *  class of the Apache Http Client `org.apache.http` library
+ */
+class TypeApacheHttpRequestBase extends RefType {
```

Comment:
Rename to `ApacheHttpRequest`. It includes subclasses, so naming it `Base` is misleading. And the `Type` prefix doesn't add much.